core/oslc4j-jena-provider/src/test/java/org/eclipse/lyo/oslc4j/provider/jena/MessageBodyReaderTest.java

```diff
@@ -0,0 +1,65 @@
+package org.eclipse.lyo.oslc4j.provider.jena;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collection;
+
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.lyo.oslc4j.core.model.OslcMediaType;
+import org.eclipse.lyo.oslc4j.provider.jena.test.resources.ResourceWithoutShape;
+import org.eclipse.lyo.oslc4j.provider.jena.test.resources.TestResource;
+import org.junit.Test;
+
+public class MessageBodyReaderTest {
```

Comment:
Need to check if we can add a simple `@OslcNotQueryResult` test here. @jadelkhoury @jamsden do you have some code that you could find that uses that annotation to help @jhemm2 write a test? I think we annotate JAX-RS methods with it IIRC.